packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -361,7 +361,7 @@ class IOSSimulator extends Device {
   @override
   bool isSupported() {
     if (!p.platform.isMacOS) {
-      _supportMessage = "Not supported on a non Mac host";
+      _supportMessage = 'Not supported on a non Mac host';
```

Comment:
Re-worded in #7859. Can't believe I missed this! *shame* 

Grepped the entire repo for 'non ', since these drive me insane as well. Turns out this was the only one.